### PR TITLE
Handle string case & add tests

### PR DESCRIPTION
**Pull request by mcqua007**
We had a few people leave issues that they didn't know they had to pass JSON.parse the response from Shopify. This change handles that case. I also added Jest to test the change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,20 +65,22 @@ convertSchemaToHtml(JSON.parse(richTextResponse), 'rich-text-wrap')
 ```
 
 Another more in depth example:
-``` javascript
-export default Test(){
+
+```javascript
+export default RenderedHTML(){
  const richTextResponse  = await getRichTextFromShopify()
```
Comment (jeffreyguenther) [inline README.md:71]: In this example, what are we saying `getRichTextFromShopify` is returning? String or JSON?

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 export function convertSchemaToHtml(arr, scoped = false) {
```
Comment (jeffreyguenther) [inline src/index.js]: Looks like a good opportunity to rename `arr` to something more semantic.

File: test/index.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/**
```
Comment (jeffreyguenther) [inline test/index.test.js:1]: Let's also add a test for the JSON object case

File: README.md (hunk 1 above)
Comment (mcqua007) [reply README.md:71]: Shopify returns a string. I just checked in our remix app using the metaobjects. So this change allows the dev to pass in either or.

File: src/index.js (hunk 2 above)
Comment (mcqua007) [reply src/index.js]: I was thinking the same thing! Have any suggestions ? maybe richText ?

File: test/index.test.js (hunk 3 above)
Comment (mcqua007) [reply test/index.test.js:1]: Good idea.

**Pull request closed without merging**